Userland/Libraries/LibWeb/CSS/GridTrackSize.cpp

```diff
@@ -116,6 +124,25 @@ String GridMinMax::to_string() const
     return MUST(builder.to_string());
 }
 
+GridFitContent::GridFitContent(GridSize max_grid_size)
+    : m_max_grid_size(max_grid_size)
+{
+}
+
+GridFitContent::GridFitContent()
+    : m_max_grid_size(GridSize::make_auto())
+{
+}
+
+String GridFitContent::to_string() const
+{
+    StringBuilder builder;
+    builder.append("fit-content("sv);
+    builder.appendff("{}", m_max_grid_size.to_string());
+    builder.append(")"sv);
+    return MUST(builder.to_string());
```

Comment:
this looks simple enough to use `String::formatted` instead:
```cpp
return MUST(String::formatted("fit-content({})", m_max_grid_size.to_string()))
```

Reply:
Fixed!